### ISSUE TITLE
[very minor bug] Vecdraws width is using ship position not camera position

### ISSUE THREAD

**Issue by Dunbaratu** [KSP-KOS/KOS#2101]
Part of the vecdraw logic includes a heuristic guess about how thick to draw the vector based on how far away it is (to make it still visible if you're doing something like drawing the vector from a distant planet).

(i.e. the further the distance away, the thicker the arrow so it's not a line thinner than a pixel).

The problem is that this distance is based on distance from the SHIP to the vector start, rather than from the CAMERA to the vector start.  The camera can be very far from the ship in the map view (i.e. hit TAB to cycle through planets - your ship is around Kerbin, but your camera is all the way out at Duna).  When the camera is far from the ship, but very near the vector (i.e your ship around Kerbin draws a vector at Duna, and your map view is centered on Duna) then the arrow is *very very* thick.  Unhelpfully so.

This should be fixed to use distance from camera as the heuristic, assuming we can read the camera position from the API (we should be able to, from Unity itself, I'd assume).

**Comment by ozin370** [comment]
Yeah, Hvacengi's camera addon accesses the camera position. It also makes it available to kOS scripts, so technically you could already do this with the addon and a script already (assuming that the width suffix of a vecdraw overrides this heuristic guess you are talking about.)
